src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -142,12 +142,65 @@ struct TimeDependentMapOptions {
     std::optional<std::variant<KerrSchildFromBoyerLindquist>> initial_values{};
   };
 
+  struct RotationMapOptions {
+    using type = Options::Auto<RotationMapOptions, Options::AutoLabel::None>;
+    static std::string name() { return "RotationMap"; }
+    static constexpr Options::String help = {
+        "Options for a time-dependent rotation map about an arbitrary axis. "
+        "Specify 'None' to not use this map."};
+
+    struct InitialAngularVelocity {
+      using type = std::array<double, 3>;
+      static constexpr Options::String help = {"The initial angular velocity."};
+    };
+
+    struct DecayTimescaleRotation {
+      using type = double;
+      static constexpr Options::String help = {
+          "The timescale for how fast the angular velocity approaches "
+          "its asymptotic value."};
+    };
+
+    using options = tmpl::list<InitialAngularVelocity, DecayTimescaleRotation>;
+
+    std::array<double, 3> initial_angular_velocity{};
+    double decay_timescale{
+        std::numeric_limits<double>::signaling_NaN()};
+  };
+
+  struct ExpansionMapOptions {
+    using type = Options::Auto<ExpansionMapOptions, Options::AutoLabel::None>;
+    static std::string name() { return "ExpansionMap"; }
+    static constexpr Options::String help = {
+        "Options for the expansion map. Specify 'None' to not use this map."};
+
+    struct InitialValues {
+      using type = std::array<double, 2>;
+      static constexpr Options::String help = {
+          "Initial value and deriv of expansion."};
+    };
```

Comment:
Same for expansion. For both the actual expansion, and the outer boundary expansion, each need an initial value and two time derivatives because of the `SettleToConstant`s.

Reply:
Added one more initial value for the 2nd derivative